src/main/java/ca/serum390/godzilla/domain/planning/Plan.java

```diff
@@ -1,5 +1,29 @@
 package ca.serum390.godzilla.domain.planning;
 
+import org.springframework.stereotype.Component;
+
+import ca.serum390.godzilla.api.handlers.InventoryHandler;
+import ca.serum390.godzilla.data.repositories.InventoryRepository;
+
+@Component
```

Comment:
Since this class is not in the scope of your PR, I suggest that you leave it unchanged

Reply:
changed. removed my modification in planning